projects/igniteui-angular/src/lib/directives/autocomplete/autocomplete.directive.ts

```diff
@@ -317,11 +330,7 @@ export class IgxAutocompleteDirective extends IgxDropDownItemNavigationDirective
         if (!this.collapsed) {
             return;
         }
-        const settings = Object.assign({}, this.settings, this.autocompleteSettings);
-        if (!settings.positionStrategy.settings.target) {
```

Comment:
This was actually a more concise/shorter check than the new settings getter. The assign handles the "if there's this setting" for you.